extensions/amp-viqeo-player/0.1/test/validator-amp-viqeo-player.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
```

Comment:
This is missing the license. Please add it here and to the `.out` file as well.

```
<!---
Copyright 2018 The AMP HTML Authors. All Rights Reserved.
 
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
 
      http://www.apache.org/licenses/LICENSE-2.0
 
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS-IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
-->
```

Reply:
fixed